backend/routes/group.py

```
@@ -0,0 +1,53 @@
+        posts = db.query(Post).filter(
+            Post.user_id.in_(user_ids),
+            Post.content.ilike(f"%{hashtag}%")
+        ).order_by(Post.created_at.desc()).all()
+
+        post_ids = [post.id for post in posts]
+
+        return UniversityPage(
+            university=university_name,
+            total_members=total_members,
+            departments=departments,
+            post_ids=post_ids
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)